client/jetpack-connect/style.scss

```diff
@@ -1096,19 +1081,6 @@ $colophon-height: 50px;
 			font-size: $font-body-small;
 		}
 
-		.auth-form__social,
-		.login__form-social {
-			.social-buttons__button {
-				border-color: var(--studio-woocommerce-purple);
-				color: var(--studio-woocommerce-purple);
-
-				&.disabled {
-					border-color: var(--studio-gray-30);
-					color: var(--studio-gray-30);
-				}
-			}
-		}
-
```

Comment:
These styles are also completely superseded by the global `.layout.is-jetpack-woo-dna-flow` style that sets CSS variables like `--color-accent` to the Woo brand purple color. The WP Components secondary button then picks up these colors automatically. This is how the social buttons now look during Woo-branded Jetpack Connect flow:

<img width="394" alt="Screenshot 2025-05-20 at 16 06 50" src="https://github.com/user-attachments/assets/e4b5d380-2624-4238-8246-122308182f2d" />